tests/python_client/testcases/test_index.py

```diff
@@ -2023,6 +2133,22 @@ def test_create_index_with_small_dim(self, dim):
                                   check_items={ct.err_code: 1,
                                                ct.err_msg: "dim out of range: [8, 32768]"})
 
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_diskann_enable_mmap(self):
+        """
+        target: enable mmap for unsupported indexes
+        method: diskann index enable mmap
+        expected: unsupported
+        """
+        c_name = cf.gen_unique_str(prefix)
+        collection_w = self.init_collection_wrap(c_name, schema=default_schema)
+        collection_w.insert(cf.gen_default_list_data())
+        collection_w.create_index(default_float_vec_field_name, ct.default_diskann_index, index_name=ct.default_index_name)
+        collection_w.alter_index(ct.default_index_name, {'mmap.enabled': True},
```

Comment:
how about turn on mmap using setting properties interface, and create these unsupported indexes? what is the behaviour?